apps/src/weblab/brambleHost.js

```diff
@@ -345,6 +345,10 @@ function uploadAllFilesFromBramble(callback) {
   });
 }
 
+function fileRefresh(callback) {
+  brambleProxy_.fileRefresh(callback);
```

Comment:
added `fileRefresh` to the bramble API in [this commit](https://github.com/code-dot-org/bramble/pull/7/commits/2515f39ba38ef0fa1ba199f573b3f09f4d0e34d3)